pages/bibliography.md

```diff
@@ -1,11 +1,16 @@
+{% include nav.html %} {% include css.html %}
+
 # Bibliography
 
 ---
-
-## Project Bibliography
-
-1. List your bibliography items here, whether historical and/or digital in nature, here.
-2. It should be in [Chicago Manual of Style](chicagomanualofstyle.org)
-3. Remember to *italicize* and **bold** as appropriate.
-4. See example below
-5. Clinton, Kevin. *Eleusis, the Inscriptions on Stone: Documents of the Sanctuary of the Two Goddesses and Public Documents of the Deme*. Vol. 1a: Text. Athens: The Archaeological Society at Athens, 2008. 
+1. Bradford, Alfred S. Flying the Black Flag: A Brief History of Piracy. Non-Series. Westport, Conn: Praeger, 2007. http://ezproxy.lib.usf.edu/login?url=http://search.ebscohost.com/login.aspx?direct=true&db=nlebk&AN=218223&site=eds-live. 
```

Comment:
Every one of these URLs should be linked for easy access of the reader, as mentioned in class.

In addition (I did not mention or even think of this), it is probably best to remove the ezproxy.lib.usf part (all the way through url=). so that it starts with https:.//search.ebscohost.com...

Because other people won't necessarily be accessing through USF. If they are, they will be redirected anyway. If not, they will be redirected to their own relevant portal. Here is an example linked URL with the corrected address...

Reply:
```suggestion
1. Bradford, Alfred S. Flying the Black Flag: A Brief History of Piracy. Non-Series. Westport, Conn: Praeger, 2007. [http://search.ebscohost.com/login.aspx?direct=true&db=nlebk&AN=218223&site=eds-live](http://search.ebscohost.com/login.aspx?direct=true&db=nlebk&AN=218223&site=eds-live). 
```